cookbooks/aws-parallelcluster-config/files/default/cloudwatch_agent/cloudwatch_agent_config.json

```diff
@@ -163,6 +164,42 @@
       ],
       "feature_conditions": []
     },
+    {
+      "timestamp_format_key": "metric",
+      "file_path": "/var/log/parallelcluster/clustermgtd.events",
+      "log_stream_name": "clustermgtd_events",
+      "schedulers": [
+        "slurm"
+      ],
+      "platforms": [
+        "amazon",
+        "centos",
+        "redhat",
+        "ubuntu"
+      ],
+      "node_roles": [
+        "HeadNode"
+      ],
+      "feature_conditions": []
+    },
+    {
+      "timestamp_format_key": "metric",
+      "file_path": "/var/log/parallelcluster/slurm_resume.events",
+      "log_stream_name": "resume_events",
```

Comment:
Do we usually keep the log stream name and the log name consistent? If that's the case, maybe `slurm_resume_events` for log stream?

Reply:
Yes.